cluster/addons/fluentd-gcp/fluentd-gcp.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: fluentd-cloud-logging
+  namespace: kube-system
+  labels:
+    k8s-app: fluentd-logging
+spec:
+  dnsPolicy: Default
+  containers:
+  - name: fluentd-cloud-logging
+    image: gcr.io/google_containers/fluentd-journal-gcp:1.0
```

Comment:
If we're gonna use a separate image, we should have a Makefile for it like we do for the existing one.